### PR TITLE
chore(i3s): fix naming

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

File: modules/i3s/src/i3s-content-loader.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import type {LoaderWithParser, LoaderOptions, LoaderContext} from '@loaders.gl/loader-utils';
+import type {LoaderWithParser, LoaderContext} from '@loaders.gl/loader-utils';
+import {I3SLoaderOptions} from './i3s-loader';
```
Comment (ibgreen) [inline modules/i3s/src/i3s-content-loader.ts:2]: `import type`?

File: modules/i3s/src/i3s-loader.ts [hunk 2]
```diff
@@ -31,12 +31,11 @@ export const I3SLoader: LoaderWithParser = {
   extensions: ['bin'],
   options: {
     i3s: {
-      loadContent: true,
       token: null,
       isTileset: 'auto',
       isTileHeader: 'auto',
-      tile: null,
-      tileset: null,
+      tileOptions: null,
```
Comment (ibgreen) [inline modules/i3s/src/i3s-loader.ts:37]: Are these options documented? If so it is a breaking change, maybe we need to still support the old ones and mark them as deprecated.

If not documented, should we document the new options?

If they are internal and not for users, prefix with underscore?

File: modules/tiles/src/tileset/traversers/i3s-tileset-traverser.ts [hunk 3]
```diff
@@ -72,8 +72,7 @@ export default class I3STilesetTraverser extends TilesetTraverser {
       ...tileset.loadOptions,
       i3s: {
         ...tileset.loadOptions.i3s,
-        isTileHeader: true,
-        loadContent: false
```
Comment (ibgreen) [inline modules/tiles/src/tileset/traversers/i3s-tileset-traverser.ts:76]: Is this a breaking change? Do we need to update docs or upgrade guide?

**Review by ibgreen — COMMENTED** [review]
If this is going in to a minor release we do need to think about backwards compatibiltiy.

Also consider docs and tests.

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 4]
```diff
@@ -54,22 +54,22 @@ const defaultContent: I3STileContent = {
 
 export async function parseI3STileContent(
   arrayBuffer: ArrayBuffer,
-  tile: I3STileOptions,
-  tileset: I3STilesetOptions,
+  tileOptions: I3STileOptions,
+  tilesetOptions: I3STilesetOptions,
   options?: LoaderOptions,
   context?: LoaderContext
 ) {
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:61]: Please add return type

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 5]
```diff
@@ -98,20 +98,20 @@ export async function parseI3STileContent(
...
 /* eslint-disable max-statements */
 async function parseI3SNodeGeometry(
   arrayBuffer: ArrayBuffer,
-  tile: I3STileOptions,
+  tileOptions: I3STileOptions,
   content: I3STileContent,
-  tileset: I3STilesetOptions,
+  tilesetOptions: I3STilesetOptions,
   options?: LoaderOptions
 ) {
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:116]: Please specify return type

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 6]
```diff
@@ -98,20 +98,20 @@ export async function parseI3STileContent(
...
 
-  return await parseI3SNodeGeometry(arrayBuffer, tile, content, tileset, options);
+  return await parseI3SNodeGeometry(arrayBuffer, tileOptions, content, tilesetOptions, options);
 }
 
 /* eslint-disable max-statements */
 async function parseI3SNodeGeometry(
   arrayBuffer: ArrayBuffer,
-  tile: I3STileOptions,
+  tileOptions: I3STileOptions,
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:112]: Should we group all the options parameters at the end of the function signature.

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 7]
```diff
@@ -98,20 +98,20 @@ export async function parseI3STileContent(
...
   }
 
-  content.material = makePbrMaterial(tile.materialDefinition, content.texture);
+  content.material = makePbrMaterial(tileOptions.materialDefinition, content.texture);
   if (content.material) {
     content.texture = null;
   }
 
-  return await parseI3SNodeGeometry(arrayBuffer, tile, content, tileset, options);
+  return await parseI3SNodeGeometry(arrayBuffer, tileOptions, content, tilesetOptions, options);
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:106]: Worth grouping options at the end?

**Review by ibgreen — APPROVED** [review]
Should we use _tileOptions and _tilesetOptions?

File: modules/tiles/src/tileset/traversers/i3s-tileset-traverser.ts (hunk 3 above)
Comment (belom88) [reply modules/tiles/src/tileset/traversers/i3s-tileset-traverser.ts:76]: I fount out that loadContent is absolutely useless option in current implementation

**Pull request merged**